ComputeDataDrivenCrossSection.py

```diff
@@ -27,13 +27,28 @@
 parser.add_argument("--FD", action='store_true', help='flag to compute FD cross section', default=False)
 parser.add_argument("--Dplus", action='store_true', help='flag to compute D+ cross section', default=False)
 parser.add_argument("--Ds", action='store_true', help='flag to compute Ds cross section', default=False)
+parser.add_argument("--Lc", action='store_true', help='flag to compute Lc cross section', default=False)
 parser.add_argument("--batch", action='store_true', help='suppress video output', default=False)
+parser.add_argument("--corr", action='store_true', help='assume raw yield and (prompt/feed-down) fraction uncertanties as correlated', default=False)
```

Comment:
do not exceed the limit of 120 characters in https://github.com/DmesonAnalysers/DmesonAnalysis/blob/master/.pylintrc